packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -32,13 +32,15 @@ import {
   NewVisitPayload,
   toOmrsIsoString,
   toDateObjectStrict,
+  useLayoutType,
 } from '@openmrs/esm-framework';
 import { amPm, convertTime12to24, DefaultWorkspaceProps } from '@openmrs/esm-patient-common-lib';
 import VisitTypeOverview from './visit-type-overview.component';
 import styles from './visit-form.component.scss';
 
-const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ isTablet, patientUuid, closeWorkspace }) => {
+const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWorkspace }) => {
   const { t } = useTranslation();
+  const isTablet = useLayoutType() === 'tablet';
```

Comment:
Looks like this isn't needed here

Reply:
It is used in line 230. (`<ButtonSet className={isTablet ? styles.tablet : styles.desktop}>`)